### PR TITLE
egctl x translate

### PR DESCRIPTION
**Pull request by arkodg**
* add an `experimental` subcommand called `translate` allowing users to observe the translated xds output when gateway api resources are inputted

* this should help users understand the mapping between gatewapi resources and xds resources which will help with https://github.com/envoyproxy/gateway/issues/24 and https://github.com/envoyproxy/gateway/issues/31

### CODE REVIEW

File: internal/cmd/egctl/experimental.go [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+import (
+	"github.com/spf13/cobra"
+)
+
+func NewExperimentalCommand() *cobra.Command {
+
+	experimentalCommand := &cobra.Command{
+		Use:     "experimental",
+		Aliases: []string{"x"},
+		Short:   "Experimental features",
```
Comment (zirain) [inline internal/cmd/egctl/experimental.go:17]: great! 

can you update the `egctl` doc, `Experimental features do not guarantee compatibility and stability.`

Comment (arkodg) [reply internal/cmd/egctl/experimental.go:17]: sure will do, will also add a user facing `egctl` doc along with this PR once ready

**Review by Xunzhuo — COMMENTED** [review]
This is great, I like this feature.

**Review by Xunzhuo — APPROVED** [review]
LGTM! Thanks!

**Pull request merged**